### ISSUE TITLE
OpenStudio HVAC objects

### ISSUE THREAD

**Issue by shorowit** [NREL/OpenStudio-BEopt#32]
### Near-term plan
- Use AirLoopHVACUnitarySystem for:
  - [x] furnaces
  - [x] central ACs
  - [x] ASHPs
- Use ZoneHVACTerminalUnitVariableRefrigerantFlow for:
  - [x] MSHPs
- [x] Note: Waiting on OS team to support multi/variable speed coils with AirLoopHVACUnitarySystem. See https://github.com/NREL/OpenStudio/issues/2172. Therefore, we can only switch over our single-speed central ACs and ASHPs at the moment. Joe already switched over our single-speed central ACs, so let's switch single-speed ASHPs as well.
- [x] When we switch over, this will eliminate the need for the BEopt workaround mentioned in https://github.com/NREL/OpenStudio-Beopt/issues/25
### Long-term plan
- [ ] Test and use variable speed coils (rather than multi-speed coils) for variable-speed ACs/ASHPs/MSHPs
- [ ] Use ZoneHVACWindowAirConditioner instead of ZoneHVACPackagedTerminalAirConditioner for room AC model? The former object is not currently available in OS.

cc @joseph-robertson @ejhw @jmaguire1

**Comment by joseph-robertson** [comment]
I updated to OS 1.12.2.  It appears that the OS team has implemented support for multi/variable speed coils with AirLoopHVACUnitarySystem.

**Comment by shorowit** [comment]
Where do we stand on these items?

**Comment by joseph-robertson** [comment]
Updated above.

**Comment by shorowit** [comment]
Will add a separate ticket specific to variable speed coils. This ticket is essentially complete.
